exec/jdbc/pom.xml

```diff
@@ -21,9 +21,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>net.hydromatic</groupId>
-      <artifactId>optiq-avatica</artifactId>
-      <version>0.9-drill-r20</version>
+      <groupId>org.apache.calcite</groupId>
+      <artifactId>calcite-avatica</artifactId>
+      <version>1.4.0-drill-r10</version>
```

Comment:
Can we use refactor the version into root pom and use the same version both in jdbc and exec/java-exec?